### PR TITLE
[Doc]Modify some para descs in flink

### PR DESCRIPTION
**Pull request by hellolilyliuyi**
## What I'm doing:

Fixes #issue

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function
- [ ] This is a backport pr

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.2
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5

### CODE REVIEW

File: docs/en/loading/Flink-connector-starrocks.md [hunk 1]
```diff
@@ -99,13 +99,13 @@ In your Maven project's `pom.xml` file, add the Flink connector as a dependency
 
 **Required**: Yes<br/>
 **Default value**: NONE<br/>
-**Description**: The address that is used to connect to the MySQL server of the FE.  You can specify multiple addresss, which must be separated by a comma (,). Format: `jdbc:mysql://<fe_host1>:<fe_query_port1>,<fe_host2>:<fe_query_port2>,<fe_host3>:<fe_query_port3>`.
+**Description**: The address that is used to connect to the MySQL server of the FE.  You can specify multiple addresses, which must be separated by a comma (,). Format: `jdbc:mysql://<fe_host1>:<fe_query_port1>,<fe_host2>:<fe_query_port2>,<fe_host3>:<fe_query_port3>`.
```
Comment (evelynzhaojie) [inline docs/en/loading/Flink-connector-starrocks.md:102]: of the FE.  You --- extra space

File: docs/en/loading/Flink-connector-starrocks.md [hunk 2]
```diff
@@ -123,20 +123,14 @@ In your Maven project's `pom.xml` file, add the Flink connector as a dependency
 
 **Required**: Yes<br/>
 **Default value**: NONE<br/>
-**Description**:  The username of the account that you want to use to load data into StarRocks.  The account needs [SELECT and INSERT privileges](../sql-reference/sql-statements/account-management/GRANT.md).
+**Description**:  The username of the account that you want to use to load data into StarRocks. The account needs [SELECT and INSERT privileges](../sql-reference/sql-statements/account-management/GRANT.md).
```
Comment (evelynzhaojie) [inline docs/en/loading/Flink-connector-starrocks.md:126]: :  The username -- extra space after the colon

Comment (evelynzhaojie) [reply docs/en/loading/Flink-connector-starrocks.md:126]: SELECT and INSERT privileges on the StarRocks table

File: docs/en/loading/Flink-connector-starrocks.md [hunk 3]
```diff
@@ -68,7 +68,7 @@ In your Maven project's `pom.xml` file, add the Flink connector as a dependency
 
 ### Compile by yourself
 
-1. Download the [Flink connector package](https://github.com/StarRocks/starrocks-connector-for-apache-flink).
+1. Download the [Flink connector source code](https://github.com/StarRocks/starrocks-connector-for-apache-flink).
```
Comment (evelynzhaojie) [inline docs/en/loading/Flink-connector-starrocks.md:71]: in Row 13, "SELECT and INSERT privileges on the target StarRocks table"

**Review by evelynzhaojie — APPROVED** [review]

**Review by amber-create — APPROVED** [review]

**Pull request merged**